interface-elements-for-desktop/articles/rich-text-editor/tables/rotate-table-cell-content.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Rotate Table Cell Content
+author: Polina Fedorova
+---
+# Rotate Table Cell Content
+
+The Rich Text Editor allows you to change the rotation type of the table cell content.
```

Comment:
The Rich Text Editor allows you to change the table cell content's rotation type.